storybook/stories/Components/MarkdownEditor.tsx

```diff
@@ -2,8 +2,6 @@ import { createComponent } from '@equinor/fusion-react-utils';
 
 import { MarkdownEditorElementProps, MarkdownEditorElement, markdownEditorTag } from '@equinor/fusion-wc-markdown';
 
-export { MenuSizes } from '@equinor/fusion-wc-markdown';
```

Comment:
why not using MenuSizes type?

Reply:
it gives warnings in storybook console. but its added back